src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,14 @@
-export const CommentInfo = () => <>Put the comment here</>;
+import React from 'react';
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <p className="CommentInfo__body">{comment.body}</p>
+    <p className="CommentInfo__name">{comment.name}</p>
+    <p>
+      <a className="CommentInfo__email" href={`mailto:${comment.email}`}>
+         {comment.email}
+      </a>
+    </p>
+  </div>
+);
```

Comment:
The structure of the component itself does not match what it was originally, which is why styles are broken.
But if it passes the tests this way, then maybe it doesn't matter. But I recommend you to look in the master branch to see what exactly is the structure of this component.

Reply:
> The structure of the component itself does not match what it was originally, which is why styles are broken. But if it passes the tests this way, then maybe it doesn't matter. But I recommend you to look in the master branch to see what exactly is the structure of this component.

Good point